modules/standard/CTypes.chpl

```diff
@@ -205,19 +206,19 @@ module CTypes {
   }
```

Comment:
Looks like the field and methods for c_ptrConst are undocumented, it'd be good to add documentation comments for them.